src/eq_schema/Answer.test.js

```diff
@@ -221,6 +221,63 @@ describe("Answer", () => {
         expect(answer.minimum.value).toEqual("2017-02-17");
       });
 
+      it("should add earliest date previous answer", () => {
+        const answer = new Answer(
+          createAnswerJSON({
+            type: "Date",
+            validation: {
+              earliestDate: {
+                id: "1",
+                enabled: true,
+                entityType: "PreviousAnswer",
+                custom: null,
+                previousAnswer: {
+                  id: "3"
+                },
+                offset: {
+                  value: 4,
+                  unit: "Days"
+                },
+                relativePosition: "Before"
+              },
+              latestDate: {
+                enabled: false
+              }
+            }
+          })
+        );
+
+        expect(answer.minimum).toMatchObject({
+          answer_id: "answer3"
+        });
+      });
+
+      it("should drop validation that has an entity type of PreviousAnswer but no answer", () => {
```

Comment:
I guess this fine for now. Going forward I wonder if we should be dropping things silently in Publisher. Feels like that should be pushed up into Author validation.

I had the same dilemma with the routing based on number publisher change. Decided not to drop the routing condition if number input wasn't provided.

Reply:
Yea.  I agree. I think it should be pushed up into Author validation.